client/me/billing-history/transactions-table.jsx

```diff
@@ -31,6 +32,21 @@ class TransactionsTable extends React.Component {
 		header: false,
 	};
 
+	renderTransactionAmount = transaction =>
+		config.isEnabled( 'show-tax' ) && transaction.tax_amount ? (
+			<Fragment>
+				<div>{ transaction.amount }</div>
+				<div className="billing-history__transaction-tax-amount">
+					{ this.props.translate( '(+%(taxAmount)s tax)', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '+%(taxAmount)s tax' )` *ES Score: 24*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).